Jenkinsfile

```diff
@@ -139,31 +45,35 @@ node {
       runTests(testImageName, BUILD_NUMBER)
```

Comment:
Is there a reason not to use `buildTestImage()` and `runTests()` from the shared library?

Reply:
Spoke to Matthew - this is because the shared library is based on the docker-compose files being written differently. Either this repo or the shared library and other repos will need updating based on which way we decide to structure those files.